src/amo/withExperiment.js

```diff
@@ -42,12 +46,17 @@ import type { DispatchFunc } from 'amo/types/redux';
  *          { id: 'variant-b', percentage: 0.2 },
  *          { id: NOT_IN_EXPERIMENT, percentage: 0.6 },
  *        ],
+ *        shouldExcludeUser(state) {
```

Comment:
I'd use a single object argument with a `state` prop here in case we want to inject other props in the future:

```suggestion
 *        shouldExcludeUser({ state }) {
```

Reply:
Yeah, I saw that in your example, but I figured we didn't need the object if all we were injecting was `state`, but good point on future-proofing it!